src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -400,6 +556,13 @@ private static byte[] SendUDPRequest(string browserHostname, int port, byte[] re
 
             const int sendTimeOut = 1000;
             const int receiveTimeOut = 1000;
+
+            IPAddress address = null;
+            if (IPAddress.TryParse(browserHostname, out address) && address.AddressFamily == AddressFamily.InterNetworkV6)
```

Comment:
It is because sending instance info request to SQL Server Browser over IPv6 has an issue, and seems to be needed to some research. Currently if an IPv6 address is given, it is converted to domain name , and SSRP is processed, which returns successful result. Why don't we separate this issue from Integrated Authentication feature?